arbitrum-docs/build-decentralized-apps/token-bridging/layer-leap.mdx

```diff
@@ -0,0 +1,84 @@
+  - The Layer 2 must be an Arbitrum chain due to the use of retryables.
+- _Future support: Layer Leap will support Orbit chains with custom gas tokens in a future release_
+
+## Implementation
+
+_How can Orbit chains adopt this feature?_
+
+**UI Adoption:** Layer leap contracts can be integrated into your own bridge or added to your chain’s UI for the Arbitrum bridge (bridge.arbitrum.io)
+
+### Arbitrum bridge UI: if you are already on the arbitrum bridge UI then you can sign up for layer leap adoption by using this [github template](https://github.com/offchainlabs/arbitrum-token-bridge/issues/new?assignees=&labels=feat%2ctriage&projects=&template=add-layerleap-request.yml&title=%5bfeat%5d%3a+enable+layer+leap+for+%3corbit+chain%3e).
+
+    - If you are not on the Arbitrum Bridge UI yet you can sign up by filling out this [GitHub Template](https://github.com/OffchainLabs/arbitrum-token-bridge/issues/new?assignees=&labels=feat%2Ctriage&projects=&template=add-orbit-chain-request.yml&title=%5Bfeat%5D%3A+), and then also fill out the template above.
+    - You can demo Layer Leap on the Arbitrum Bridge by bridging to [Rari](https://bridge.arbitrum.io/?destinationChain=rari-mainnet&sourceChain=ethereum) and [Proof of Play](https://bridge.arbitrum.io/?destinationChain=pop-apex&sourceChain=ethereum).
+
+### Custom bridge UI: if you want to integrate layer leap functionality into your custom bridge UI, read the [info page on the layer leap github](https://github.com/offchainlabs/l1-l3-teleport-contracts/blob/main/docs/info.md) to learn more.
+
+_What chains are currently supported?_
+
+Layer Leap requires an L1 teleporter contract for each L2 chain. `L1Teleporter` contracts are deployed on Ethereum L1 for Arb One and Arb Nova.
+
+| Contract              | Address                                                                 |
+| --------------------- | ----------------------------------------------------------------------- |
+| Arb One L1Teleporter  | https://etherscan.io/address/0xCBd9c6e310D6AaDeF9F025f716284162F0158992 |
+| Arb Nova L1Teleporter | https://etherscan.io/address/0xCBd9c6e310D6AaDeF9F025f716284162F0158992 |
+
+For non Arbitrum One L3s:
+
+- To enable Layer Leap for your Layer 3 you can follow the Deployment Procedure defined [here](https://github.com/OffchainLabs/l1-l3-teleport-contracts/blob/main/docs/info.md). Before self deploying, let Offchain Labs know about your request!
+
+_Implementation requirements for RaaSes?_
+
+Sending tokens from L2 to L3 relies on creation of a retryable. If you are supporting Layer Leap for your Orbit chains it’s highly recommended to be running the Retryables Monitoring tool: https://github.com/OffchainLabs/arb-retryables-monitoring. This will allow you to track and redeem any failed retryables
+
+_What configuration options or parameter changes should I consider?_
```

Comment:
This feels incomplete and I'd argue that, even if it were complete, this info should be available before we give the user a way to adopt it